hello-typescript/src/1-reverse-array.ts

```diff
@@ -0,0 +1,9 @@
+export function reverseArray<T>(items: T[]): T[] {
+    const reversedArray: T[] = [];
+
+    for (let i: number = 0; i < items.length; ++i) {
+        reversedArray[i] = items[items.length - i - 1];
+    }
```

Comment:
بهتر بود از آخر به اول حلقه بزنید که خواناتر و واضح‌تر باشه

```suggestion
    for (let i = items.length - 1; i >= 0; i--) {
        reversedArray[i] = items[i];
    }
```